src/ds5/advanced_mode/presets.cpp

```diff
@@ -444,8 +444,8 @@ namespace librealsense
         p.hdad.ignoreSAD = 0;
         p.amplitude_factor.amplitude = 0.f;
     }
-
-    void default_450(preset& p)
+    // the only different between high res to mid & low is the amplitude_factor value
+    void default_450_high_res(preset& p)
```

Comment:
Maybe for high res first load the lowres preset and then adjust only the required fields.
i think it will simplify maintenance and prevent C&P issues

Reply:
done